sdk/deviceupdate/azure-iot-deviceupdate/src/samples/java/com/azure/iot/deviceupdate/GetUpdateNotFoundSample.java

```diff
@@ -0,0 +1,30 @@
+package com.azure.iot.deviceupdate;
```

Comment:
same here for copyright. easier to config your IDE to do it for you.

curious why this is not caught by checkstyle.

Reply:
I found checkstyle is skipped on main branch: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/deviceupdate/azure-iot-deviceupdate/pom.xml#L33
That's why the copyright error is not checked.
I will enable it in this pr.